### PR TITLE
School library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by edubew**
### In this module, the following was done:
- Add and configure linter files
- Create a class Person with the following:
     - Constructor with name, age, and parent_permission as a parameter. name and parent_permission are optional and have default values of "Unknown" and true.
     - Use Getters and Setters method for @id, @name, and @age.
     - Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
     - Public method can_use_services? returns true if a person is of age or if they have permission from parents.
- Create class Student with the following:
     - Inherits from Person.
     - Constructor extends parent's constructor by adding @classroom and a parameter for it.
     - Method play_hooky that returns "¯\(ツ)/¯".
- Create a class Teacher with the following:
     - Inherits from Person.
     - The constructor extends the parent's constructor by adding @specialization and a parameter for it.
     - Override can_use_services? so it always returns true.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+A heartfelt thank you to Microverse for their invaluable support, guidance, and the incredible opportunity they have provided us. Without them, this project would not have been possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/edubew/School-library/blob/dev/LICENSE) licensed.
```
Comment (cynthiainga) [inline README.md]: - [x] Please kindly fix the MIT link, your link is broken and when I click on it I got an error. Kindly fix that 🙏🏾

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name: 'Unknown')
+    super(id, age, name)
```
Comment (cynthiainga) [inline teacher.rb]: - [x] You forgot to pass the `parent_permission` as a parameter in `initialize` and in super`, please kindly fix that 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @edubew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Each class has its own `.rb` file ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
Hi @edubew 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the requested changes 👏🏾 
- Good PR description 👌🏽 
- Descriptive README file 🙂 

# Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**